eclass/qt5-build.eclass

```diff
@@ -74,16 +81,15 @@ case ${PV} in
 	*_alpha*|*_beta*|*_rc*)
 		# development release
 		QT5_BUILD_TYPE="release"
-		MY_P=${QT5_MODULE}-everywhere-src-${PV/_/-}
-		SRC_URI="https://download.qt.io/development_releases/qt/${PV%.*}/${PV/_/-}/submodules/${MY_P}.tar.xz"
-		S=${WORKDIR}/${MY_P}
+		_QT5_P=${QT5_MODULE}-everywhere-src-${PV/_/-}
```

Comment:
maybe you can use this definition for the default assignment above? since the stable versions won't have any `_` in `PV`, the substitution `${PV/_/-}` should be a noop.

or maybe just drop the whole "development release" case? I suppose there won't be any more